src/worker.js

```diff
@@ -238,6 +238,12 @@ const aioli = {
 		return aioli._fileop("download", path);
 	},
 
+	downloadBlob(path) { // return a blob instead of a URL, so it can be downloaded with other files by jszip
+		const file = aioli.fs.readFile(path);
+        const blob = new Blob([ file ]);
+        return blob;
```

Comment:
Can you update this to use the same structure used by the other file system functions? i.e. it should `return aioli._fileop(...)`, and you can update the `_fileop` function to contain the logic here.